core/lib/src/unix.rs

```diff
@@ -0,0 +1,100 @@
+use crate::data::NetStream;
+use crate::http::hyper::net::NetworkStream;
```

Comment:
Do we need any of the `cfg(unix)` attributes in here? I believe this entire module is only included when `cfg(unix)`.